src/Conference/AddConferenceForm.jsx

```diff
@@ -82,8 +83,9 @@ const Form = () => {
         required
         maxLength="20"
       />
-      <Input type="submit" value="submit" />
+      <Input type="submit" value="Submit" />
     </form>
   );
 };
-export default Form;
+export default withCookies(AddConferenceForm);
+export { AddConferenceForm as WrappedAddConferenceForm };
```

Comment:
Specifically the default is wrapped with cookies HoC, here you are exporting the unwraped with the name wrapped?

Reply:
Have changed the exports for AddConferenceForm and ConferenceDetails: prefixed the main one with the word 'Cookie', then exported as default; and exported the cookie-less one inline for test usage.